test/run_fuse_test.go

```diff
@@ -30,15 +31,32 @@ func createEngine() Engine {
 	return e
 }
 
-func createUserFuse(t testing.TB, ith int, config *libkbfs.ConfigLocal,
-	opTimeout time.Duration) *fsUser {
+func createUserFuse(t testing.TB, ith int, username libkb.NormalizedUsername,
+	config *libkbfs.ConfigLocal, opTimeout time.Duration) *fsUser {
+	ctx := context.Background()
+	ctx, cancelFn := context.WithCancel(ctx)
+	createUserSuccess := false
+	defer func() {
+		if !createUserSuccess {
+			cancelFn()
+		}
+	}()
+
+	filesys := libfuse.NewFS(config, nil, false)
+
+	ctx = filesys.WithContext(ctx)
+	logTags := logger.CtxLogTags{
+		CtxUserKey: CtxOpUser,
+	}
+	ctx = logger.NewContextWithLogTags(ctx, logTags)
+	ctx = context.WithValue(ctx, CtxUserKey, username)
+
 	log := logger.NewTestLogger(t)
 	debugLog := log.CloneWithAddedDepth(1)
 	fuse.Debug = func(msg interface{}) {
-		debugLog.Debug("%s", msg)
+		debugLog.CDebugf(ctx, "%s", msg)
```

Comment:
I'm not sure about this -- isn't `fuse.Debug` basically a global, so it will print out the username of whoever called this last for everyone's log messages?